js/src/Ice/UserException.js

```diff
@@ -5,11 +5,6 @@
 import { Exception } from "./Exception.js";
 
 export class UserException extends Exception {
-    constructor() {
```

Comment:
Instead of inheriting Exception's ctor, I think UserException should define a `()` constructor. Derived user exceptions should not support setting the error message (and more) at construction time.